### PR TITLE
Manual changes

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Manual changes from #6589 
- Removes unused constants
- Moves the constants in StructureTagConstants to SchematicTagConstants
- Renames the constant LEATHER_LEVEL_RANGE to LEATHER_BUILDING_LEVEL_RANGE to fit the other constants
- Fixes some typos in jdocs and todos
- Bumps JEI, Structurize, and Forge versions
- Removes already-completed todo
- Completes TODO for the lumberjack "give tool" button
- Removes some dead lang strings (relating to the scan tool)
- Removes some dead code
- some research and mystical site lang fixes

Review please

### CODE REVIEW

File: src/api/java/com/minecolonies/api/util/InventoryUtils.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
      */
     private InventoryUtils()
     {
-        /*
```
Comment (Asherslab) [inline src/api/java/com/minecolonies/api/util/InventoryUtils.java]: these lines aren't javadoc. they should remain as they were. IDE even gives a warning about it

**Review by Asherslab — DISMISSED** [review]
i don't see anything glaringly wrong.
If it compiles i dont care. 

Although, why are we bumping forge/jei versions? shouldn't do that unless we *need* to

**Comment by ravenbuilder934** [comment]
Why not?

**Comment by ravenbuilder934** [comment]
ah I see the problem with forge

**Comment by Nightenom** [comment]
> i don't see anything glaringly wrong.
> If it compiles i dont care.
> 
> Although, why are we bumping forge/jei versions? shouldn't do that unless we _need_ to

imho We should bump forge regularly to ensure that ppl can run latest versions

**Comment by Asherslab** [comment]
@Nightenom 
forge shouldn't be pushing breaking changes. if they do we can bump then. And they're pretty vocal about it when they do
but building for the lowest version we can means that players can use any version above that (as long as there is no breaking change)

**Review by marchermans — DISMISSED** [review]

**Pull request merged**